lighthouse-core/audits/dobetterweb/uses-optimized-images.js

```diff
@@ -114,7 +117,9 @@ class UsesOptimizedImages extends Audit {
 
     let displayValue = '';
     if (totalWastedBytes > 1000) {
-      displayValue = `${Math.round(totalWastedBytes / KB_IN_BYTES)}KB potential savings`;
+      const totalWastedKb = Math.round(totalWastedBytes / KB_IN_BYTES);
+      const totalWastedMs = Math.round(totalWastedBytes / networkThroughput * 1000);
+      displayValue = `${totalWastedKb}KB (~${totalWastedMs}ms) potential savings`;
```

Comment:
Because this number is super hand-wavy let's reduce the precision some more. Shall we round to the nearest ten?  e.g. 497 => 500 and  432 => 430.